meeting-notes/CONTRIBUTING.md

```diff
@@ -0,0 +1,203 @@
+- **Label**: Please choose the relevant label for your issue. If no label is relevant, leave this blank and leave a comment in your issue (see below) with the proposed label. The CMWG will then add this to your issue.
+
+7. Once you have added all the details, select `Create`
+
+![Create issue](../images/create_issue.png "Create issue")
+
+8. Congrats! Your issue will now appear in the `Ideas` column of the project board.
+
+![Issue added](../images/issue_added.png "Issue added")
+
+The issue will be discussed at the next CMWG meeting, and if approved move into backlog.
+
+#### Progressing an issue
+
+Once an issue is in Backlog, anybody from the community can take it on! If you take on an issue we ask that you:
+1. **Assign yourself to it** using the `Assignees` function
+2. **Add any updates in comments in the issue**
+3. **Move it into the relevant columns**. For instance, if you have started work on it, you should move the issue to the `In progress` column.
+
+### Contributing through our Google Form
+
+You’re also able to contribute to our [project board](https://github.com/orgs/uk-tre/projects/1/views/1) through this [Google Form](https://forms.gle/ibVkn5MuuprN7ZCAA).
+
+This Google Form asks for the same information as our template issues, namely:
+- **Title**: A meaningful title for the issue. Try to begin it with an active verb.
+- **Detail**: Any detail and context around your issue it would be good for others to know
+- **Actions**: A bulletpoint list of what needs to be done
+- **Who can help**: A list of people who can help.
+
+This form will come through to the Community Management Working Group, who will review it. 
+If approved, it will be added as an issue in the `Ideas` column of our project board. 
+If they have any further questions, they’ll reach out to you.
+
+Further interaction will take place on the GitHub project board - you can see details on how to interact in the `Contributing directly through GitHub` section above.
+
+### Working Group meetings
+
+The Community Management Working Group meetings follow the same Sprint-inspired structure, outlined below:
+
+1. Anybody raises any pressing issues that need immediate discussion with the group
+2. The group gets progress reports on all `Blocked` and `In Progress` tasks, and decides next steps as required
+3. The group talks through `Ideas` and either approves or rejects them (providing reasons, if approved they move it into `Backlog`, if rejected the reasons communicated with the person who opened the issue.
+4. The group reviews `Planned` and moves items into `In progress`, `Backlog` or keeps them in `Planned` as required.
+5. The group reviews the `Backlog` and moves anything planned to be started in the upcoming week into `Planned`.
+6. AOB.
+
+## ♻️ License
+
+This work is licensed under the **MIT license (code) and Creative Commons Attribution 4.0 International license (for documentation)**. 
```

Comment:
```suggestion
This work is licensed under the **MIT license**. 
```
The website (which is our main public content) is MIT https://github.com/uk-tre/website/blob/main/LICENSE.txt
I think we should stick with MIT for now, and change it later if we want to